### DISCUSSION TITLE
GSOC'23: KerasCV & KerasNLP: Creating Tutorials

### DISCUSSION THREAD

**Discussion by NiharJani2002** [keras-team/keras-cv#1510] [Ideas]
The Main Purpose of a Tutorial is to teach someone without being there as a person. I assume, the learners either will be **Newbies** or **Intermediates**  or  **Experts** who are trying to find something new.

**Comment by IMvision12** [comment]
Hello, @NiharJani2002 I suppose you shouldn't publicly share your timeline as part of your GSoC. If you need a mentor to evaluate your proposal, send it to [tensorflow-gsoc@google.com].

**Comment by NiharJani2002** [comment]
> The Main Purpose of a Tutorial is to teach someone without being there as a person. I assume, the learners either will be **Newbies** or **Intermediates**  or  **Experts** who are trying to find something new.

**Reply by NiharJani2002** [reply]
Okay
